<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWidget</class>
 <widget class="QWidget" name="SimulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushBtn_SimulateStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>仿真开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushBtn_SimulatePause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>仿真暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushBtn_SimulateStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>仿真停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_TaskClear">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>清空任务</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushBtn_add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushBtn_del">
           <property name="text">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushBtn_addToList">
           <property name="text">
            <string>添加到列表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_open">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_CreatTaskToFile">
       <property name="text">
        <string>生成任务列表到文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
